cmd/containerd/server/server.go

```diff
@@ -88,6 +88,15 @@ func CreateTopLevelDirectories(config *srvconfig.Config) error {
 	if err := sys.MkdirAllWithACL(config.State, 0o711); err != nil {
 		return err
 	}
+	if config.State != defaults.DefaultStateDir {
+		// XXX: socketRoot in pkg/shim is hard-coded to the default state directory.
```

Comment:
I've never seen XXX before, but apparently it's an [old Java convention](https://www.oracle.com/java/technologies/javase/codeconventions-programmingpractices.html#395).

(no action required here)